README.md

```diff
@@ -104,14 +104,17 @@ Recommended: Add `./node_modules/.bin` to your `PATH`.  This will allow you to r
 
 ### Set Up Environment Variables
 
-Read the `ENV_CHANGELOG.md` file to see what environment variables have been added, dropped, or changed.
-Copy the file [example/dotenv](example/dotenv) to a file named `.env` in the root project directory.  Follow the instructions in that file to populate any blank variables.  Variables that have values in the example can be kept as-is, as they provide good defaults for local development.  To check environment differences between the example and local, run:
+Harmony uses environment variables for managing much of its configuration. Most of the variables can be defaulted, and harmony provides those defaults suitable for local development in the `env-defaults` file. In order to set up the remaining variables, run the following from the harmony project root:
 
 ```
-$ git diff --no-index .env example/dotenv
+$ bin/create-dotenv
 ```
 
-We recommend doing this any time you receive an example/dotenv update to ensure there are no new variables needed.
+The script will create a file named `.env` in the root project directory containing only those parameters that cannot be defaulted. Open the file and update the values for any of the variables that are currently blank. Detailed information for the environment variables can be found in the `env-defaults` file.
```

Comment:
I think this is all good info to have captured, but noting for future PR that we can probably skip it in a quick start.  I'm also wondering if maybe our README is starting to become what most projects put in their GitHub wikis.  Anyways, no action.